### PR TITLE
Add README description for Gradle Build Server

### PR DESCRIPTION
**Pull request by Sidhaarthsr**
Extended description for the README.md file for Gradle Build Server in Buildship

### CODE REVIEW

**Review by oleg-nenashev — COMMENTED** [review]
Just a few minor comments

File: docs/events/gsoc/2024/gradle-build-server-devx.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
+
 # GSoC 2024. Gradle Build Server - DevX and Language Support in Buildship
 
-Description is coming soon
+- **Project GitLab** - https://gitlab.eclipse.org/eclipsefdn/emo-team/gsoc-at-the-ef/-/issues/5
```
[the remark this answers is not in the document]
Comment (Sidhaarthsr) [reply docs/events/gsoc/2024/gradle-build-server-devx.md]: I have added placeholders for issues for implementation that will be created in the Eclipse Buildship and BSP Gradle repositories. Should I go ahead and create those tickets right away or create them during the coding phase while we start the implementation?

**Review by oleg-nenashev — APPROVED** [review]
Thanks! Normally I recommend commenting out placeholder links, but let's do it in a follow-up shortly

File: docs/events/gsoc/2024/gradle-build-server-devx.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (oleg-nenashev) [reply docs/events/gsoc/2024/gradle-build-server-devx.md]: Yep, please can go ahead and create them. It is good for facilitating feedback from the community

**Pull request merged**